FixedPopupPanels/FixedPopupPanels.js

```diff
@@ -53,11 +54,13 @@ const fixedPopupPanelsHandlers = {
  * @ui
  * @public
  */
-const FixedPopupPanels = (props) => {
-	const handlers = useHandlers(fixedPopupPanelsHandlers, props);
-
-	return <FixedPopupPanelsBase {...props} {...handlers} />;
-};
+const FixedPopupPanels = I18nContextDecorator(
+	{rtlProp: 'rtl'},
+	(props) => {
+		const handlers = props.rtl ? null : useHandlers(fixedPopupPanelsHandlers, props);
+		return <FixedPopupPanelsBase {...props} {...handlers} />;
+	}
```

Comment:
Let's make this component `FixedPopupPanelsBase` and create jsdoc for it.

Reply:
I fixed FixedPopupPanelsDecorator to include I18nContextDecorator, and applied it to FixedPopupPanelsBase.